From d96a27257d20c68894f77bfde158ccf4a7f46b27 Mon Sep 17 00:00:00 2001
From: Bian Jin chen <kenjc.bian@rock-chips.com>
Date: Fri, 24 Nov 2017 09:04:24 +0800
Subject: [PATCH] ARM: dts: rk3126c-bnd: default to close verify boot.

Because the performance of rk3126c is less than required,
the default is closed.

Change-Id: Idf80786d00e01cebfd6d2f68a991880733483862
Signed-off-by: Bian Jin chen <kenjc.bian@rock-chips.com>
---
 arch/arm/boot/dts/rk3126-bnd-d708.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk3126-bnd-d708.dts b/arch/arm/boot/dts/rk3126-bnd-d708.dts
index baba3ba12227..366b594adae6 100644
--- a/arch/arm/boot/dts/rk3126-bnd-d708.dts
+++ b/arch/arm/boot/dts/rk3126-bnd-d708.dts
@@ -122,14 +122,14 @@
 					dev = "/dev/block/rknand_system";
 					type = "ext4";
 					mnt_flags = "ro,barrier=1,inode_readahead_blks=8";
-					fsmgr_flags = "wait,verify";
+					fsmgr_flags = "wait";
 				};
 				vendor {
 					compatible = "android,vendor";
 					dev = "/dev/block/rknand_vendor";
 					type = "ext4";
 					mnt_flags = "ro,barrier=1,inode_readahead_blks=8";
-					fsmgr_flags = "wait,verify";
+					fsmgr_flags = "wait";
 				};
 			};
 		};
-- 
2.35.3

